Makefile

```diff
@@ -1,13 +1,17 @@
 all: app editor web
 
+ifeq ($(DOCKER_REPOSITORY),)
+    DOCKER_REPOSITORY := quay.io/azavea
```

Comment:
I'm not sure that the default is necessary here; this script is only used in local development, so there shouldn't be any need for the prefix in most cases.